pandas/tests/io/test_stata.py

```diff
@@ -1310,8 +1310,7 @@ def test_unsupported_datetype(self):
 
     def test_repeated_column_labels(self):
         # GH 13923
-        msg = (r"Value labels for column ethnicsn are not unique\. The"
-               r" repeated labels are:\n-+\nwolof")
+        msg = r"The repeated labels are:\n-+\nwolof"
```

Comment:
Would it be possible check more of the error message?  AFAICT, this modified test would pass independently of the changes you made to the error message above.